tests/db/test_utils.py

```diff
@@ -570,8 +585,24 @@ def test_signed_in_multipart_mixed(self):
         """
         s = self._make_signed()
         m = email.mime.multipart.MIMEMultipart('mixed', None, [s])
-        m = utils.decrypted_message_from_file(io.StringIO(m.as_string()))
-        self.assertIn(utils.X_SIGNATURE_VALID_HEADER, m)
+        mb = m.as_bytes(policy=email.policy.SMTP)
+        m = utils.decrypted_message_from_bytes(mb)
+        self.assertEqual(m[utils.X_SIGNATURE_VALID_HEADER], 'True')
+        self.assertIn(utils.X_SIGNATURE_MESSAGE_HEADER, m)
+
+    def test_signed_in_multipart_mixed_other_mua(self):
+        """It is valid to encapsulate a multipart/signed payload inside a
+        multipart/mixed payload, verify that works.
+
+        The signature being sensitive to the way the multipart message
+        was assembled (with blank lines between parts, ...),
+        we need to make sure that we're able to validate messages generated
+        by other MUAs as well.
+        """
+        with open('tests/static/mail/protonmail-signed.eml', 'rb') as f:
+            mb = f.read()
```

Comment:
Consider using [pathlib](https://docs.python.org/3/library/pathlib.html).
```suggestion
        mb = Path('tests/static/mail/protonmail-signed.eml', 'rb').read_bytes()
```

Reply:
Amended.